packages/restitution/src/Restitution.tsx

```diff
@@ -21,7 +21,7 @@ const RestitutionValues = ({ values }: Pick<RestitutionProps, 'values'>) => {
 
 type RestitutionProps = ComponentPropsWithoutRef<'dl'> & {
   label: string;
-  value?: string;
+  value?: string | ReactNode;
```

Comment:
ReactNode possède déjà le type string donc ça fait doublon
```suggestion
  value?: ReactNode;
```

Reply:
C'est fait, merci pour ta remarque.